frontend/src/pages/api/stats/total-deposit-amount.js

```diff
@@ -2,9 +2,16 @@ import StatsApi from "../../../libraries/explorer-wamp/stats";
 
 export default async function (req, res) {
   try {
-    const totalDepositAmount = await new StatsApi(req).totalDepositAmount();
-    if (totalDepositAmount) {
-      res.send(totalDepositAmount);
+    const depositAggregatedByDate = await new StatsApi(
```

Comment:
Since this is not `.csv`-prefixed variant of the handler, we should just return JSON. Also, we may just drop the method altogether since I only created one when we did not have charts.

Reply:
will drop the folder.